packages/frontend/src/pages/home/Home.tsx

```diff
@@ -65,6 +66,9 @@ export function Home(props: HomeProps) {
               formatHashLong(update.hash),
               formatRelativeTime(update.timestamp),
               update.positionCount.toString(),
+              update.forcedTxsCount > 0
+                ? update.forcedTxsCount.toString()
+                : '-',
```

Comment:
It's absolutely fine to show 0

Reply:
It's just the design
<img width="103" alt="Screenshot 2022-06-03 at 10 01 31" src="https://user-images.githubusercontent.com/38423054/171813830-768497ec-175d-445b-8842-aad58ec8fad1.png">